javatests/arcs/core/allocator/AllocatorTestBase.kt

```diff
@@ -172,13 +147,13 @@ open class AllocatorTestBase {
 
     @Test
     fun allocator_verifyStorageKeysCreated() = runAllocatorTest {
-        writeAndReadPersonPlan.particles.forEach {
+        PersonRecipePlan.particles.forEach {
             it.handles.forEach { (_, connection) ->
                 assertThat(connection.storageKey).isInstanceOf(CreateableStorageKey::class.java)
             }
         }
         val arcId = allocator.startArcForPlan(
-            "readWritePerson", writeAndReadPersonPlan
+            "readWritePerson", PersonRecipePlan
```

Comment:
Maybe change the arcs manifest to `recipe Person` instead of `recipe PersonRecipe`? Then the variable will be `PersonPlan`. 
Certainly not an essential change.

Reply:
I could do that, I remember Piotr point that out

Reply:
Done